python/ray/util/sgd/torch/training_operator.py

```diff
@@ -1,14 +1,26 @@
 import inspect
 import logging
+from copy import copy
+import numpy as np
+import pytorch_lightning as ptl
+import torch
+import torch.nn as nn
 import os
 import tempfile
 
 import torch
 import torch.nn as nn
 from filelock import FileLock
+from pytorch_lightning import EvalResult
+from pytorch_lightning.core.step_result import Result
+from pytorch_lightning.trainer.model_hooks import TrainerModelHooksMixin
+from pytorch_lightning.trainer.optimizers import TrainerOptimizersMixin
+from pytorch_lightning.trainer.training_loop import TrainerTrainLoopMixin
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.memory import recursive_detach
```

Comment:
all of this should exist in a separate file - could you please move this? (and also avoid a hard pytorch_lightning dependency)